client/app/lib/components/IntlNotificationBar.jsx

```diff
@@ -1,26 +1,29 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { injectIntl, intlShape } from 'react-intl';
+import isString from 'lodash/isString';
+import { FormattedMessage } from 'react-intl';
 
 import NotificationBar from 'lib/components/NotificationBar';
 
 export const notificationShape = PropTypes.shape({
-  message: PropTypes.shape({
-    id: PropTypes.string.isRequired,
-  }),
+  message: PropTypes.oneOfType([
+    PropTypes.shape({
+      id: PropTypes.string.isRequired,
```

Comment:
We already have a NotificationBar component under for normal strings. Should you use that instead? I am ok if you can combine these two components into one.